### ISSUE TITLE
Benchmarks

### ISSUE THREAD

**Issue by thellimist** [PerfectlySoft/Perfect#6]
It'd be great to see benchmarks compared to other languages and frameworks.

**Comment by Danappelxx** [comment]
:+1: I would be most interested in how it does with Apache Bench (concurrency and response times).

**Comment by josephglanville** [comment]
It would also be nice to see performance of FastCGI vs the native HTTP server.

**Comment by conradwt** [comment]
I would be interesting seeing these Perfect Library benchmarks at the following location:

https://www.techempower.com/benchmarks/

**Comment by kjessup** [comment]
I generated some benchmarks and posted them to the wiki:

https://github.com/PerfectlySoft/Perfect/wiki/Benchmarks

I tested:
1. Perfect Server HTTP serving static files.
2. Apache serving static files in order to compare with Perfect Server HTTP.
3. Perfect Server HTTP serving a simple "do nothing" mustache template file.
4. Apache with mod_perfect connecting to Perfect Server FastCGI serving a simple "do nothing" mustache template file.

I posted the average requests per second as well as full ab output for a typical run.
